src/types/metadata.ts

```diff
@@ -83,7 +88,7 @@ export interface TokenMetadata {
   tokenIconOverride?: string;
   tokenSymbolOverride?: string;
   tokenNameOverride?: string;
-  localization: Record<string, unknown>;
+  localization: Record<string, Localization>;
```

Comment:
It feels confusing that a `localization` property is in fact a record of a `string` with a `Localization`.

Perhaps something like
```suggestion
  localization: Record<string, LocalizationDetails>;
```

Reply:
agree. other option can be `LocalizedProperties`

Reply:
Makes sense!

Reply:
fixed on d53eb2fc1b404ad58a1f6df757b73691fbf57a05